### PR TITLE
Restore loading of dotenv file in AEABuilder.from_aea_project

### PR DESCRIPTION
**Pull request by marcofavorito**
## Proposed changes

Restore loading of dotenv file in AEABuilder.from_aea_project

## Fixes

Addresses #1512 

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [X] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that code coverage does not decrease.
- [ ] I have checked that the documentation about the `aea cli` tool works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

n/a

### CODE REVIEW

File: aea/aea_builder.py [hunk 1]
```diff
@@ -87,6 +87,7 @@
 PathLike = Union[os.PathLike, Path, str]
 
 _default_logger = logging.getLogger(__name__)
+DEFAULT_ENV_DOTFILE = ".env"
```
Comment (DavidMinarsch) [inline aea/aea_builder.py:90]: We need to make this configurable in aea-config.yaml. Let;'s open an issue

Comment (marcofavorito) [reply aea/aea_builder.py:90]: https://github.com/fetchai/agents-aea/issues/1811

File: aea/aea_builder.py [hunk 2]
```diff
@@ -1338,6 +1339,8 @@ def from_aea_project(
         )
         builder = AEABuilder(with_default_packages=False)
 
+        load_env_file(str(aea_project_path / DEFAULT_ENV_DOTFILE))
```
Comment (marcofavorito) [inline aea/aea_builder.py:1342]: notice that we don't need to check the existence of the dotenv file. The library function `dotenv.load_dotenv` will handle that.

**Review by DavidMinarsch — APPROVED** [review]
LGTM

**Pull request merged**